packages/web/src/components/drawer/Drawer.module.css

```diff
@@ -17,9 +17,10 @@
 
 .fullDrawer {
   top: 0px;
-  height: 110vh;
+  height: 100%;
```

Comment:
Did you mean 100vh? Quite different behavior between the two.

Reply:
Mistake! Thanks! this wasn't ready for rev yet :)